docs/site/pages/privacy-policy.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Privacy Policy of Player
+---
+
+# Privacy Policy of Player
+Player operates the https://player-ui.github.io/latest/getting-started website, which provides the SERVICE.
+
+This page is used to inform website visitors regarding our policies with the collection, use, and disclosure of Personal Information if anyone decided to use our Service, the Player website.
+
+If you choose to use our Service, then you agree to the collection and use of information in relation with this policy. The Personal Information that we collect are used for providing and improving the Service. We will not use or share your information with anyone except as described in this Privacy Policy.
+
+The terms used in this Privacy Policy have the same meanings as in our Terms and Conditions, which is accessible at https://player-ui.github.io/latest/getting-started, unless otherwise defined in this Privacy Policy.
+
+## Information Collection and Use
+For a better experience while using our Service, we may require you to provide us with certain personally identifiable information, including but not limited to your name, phone number, and postal address. The information that we collect will be used to contact or identify you.
```

Comment:
We don't though?

Reply:
i was hesitant to take this out but wanted some opinions on it; we dont collect any of this information but what are the standards since this data _can_ get passed to us. I was thinking it was just standard for privacy policies to include this

Reply:
AFAIK this would only apply if we collected metrics had beacons, etc that were sent back to a service we owned

as all the devtools content stays in local storage/memory this shouldnt be relevant